docs/jwst/pixel_replace/index.rst

```diff
@@ -0,0 +1,13 @@
+.. _pixel_replace_step:
+
+===============
+Pixel Replacement
+===============
+
+.. toctree::
+   :maxdepth: 2
+
+   main.rst
+   arguments.rst
+
```

Comment:
could add a ref files section that just says "this step doesn't use any ref files", just to make it explicit (and is what we've done for other steps)

Reply:
Added.